<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="2" column="0">
     <widget class="iTableView" name="mainView">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Course :</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1">
             <widget class="QComboBox" name="HyppoCombo"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="hyppoLabel">
              <property name="text">
               <string>Hyppo.</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDateEdit" name="dateEdit">
              <property name="locale">
               <locale language="French" country="France"/>
              </property>
              <property name="displayFormat">
               <string>dd-MM-yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="CourseLabel">
              <property name="text">
               <string>Course</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="RaceCombo"/>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="BShowHideRes">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>82</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: 1px solid #716404;
    border-radius: 6px;
	color: #313131;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
                                      stop: 0 #ffd70f, stop: 1 #beaa0e);
    min-width: 80px;
}

QPushButton:selected {
	color: #313131;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #bae10d, stop: 1 #9cbd0a);
}

QPushButton:checked {
	color: #313131;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #bae10d, stop: 1 #9cbd0a);
}</string>
              </property>
              <property name="text">
               <string>Montrer/Cacher</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Arrivee</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="DateLabel">
              <property name="text">
               <string>Date</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Informations :</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="LRaceId">
              <property name="text">
               <string>Race ID :  </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ERaceId">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="LDepart">
              <property name="text">
               <string>Depart à :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EDepart">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="LPiste">
              <property name="text">
               <string>Piste :       </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EPiste">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="LPrix">
              <property name="text">
               <string>Prix :         </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EPrix">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Epreuve :</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="LType">
              <property name="text">
               <string>Type :         </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EType">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="LDIscipline">
              <property name="text">
               <string>Discipline :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EDiscipline">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="LSpecialite">
              <property name="text">
               <string>Specialité :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ESpecialite">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="LDivision">
              <property name="text">
               <string>Division :    </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EDivision">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Conditions :</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="LDistance">
              <property name="text">
               <string>Distance :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EDistance">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="LCorde">
              <property name="text">
               <string>Corde :     </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ECorde">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="LTerrain">
              <property name="text">
               <string>Terrain :   </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ETerrain">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="LLien">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Lien Ext. :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ELien">
              <property name="text">
               <string/>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Eligibles :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="EEligibles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Details :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="EDetails">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="RunnersMenue">
    <property name="title">
     <string>Participants</string>
    </property>
    <addaction name="separator"/>
    <addaction name="HistoryTrigger"/>
    <addaction name="CompareTrigger"/>
   </widget>
   <addaction name="RunnersMenue"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="HistoryTrigger">
   <property name="text">
    <string>Details &amp; historique</string>
   </property>
  </action>
  <action name="CompareTrigger">
   <property name="text">
    <string>Comparer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>iTableView</class>
   <extends>QTableView</extends>
   <header>itableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
